### DISCUSSION TITLE
23.09.27 데일리 스크럼

### DISCUSSION THREAD

**Discussion by minju412** [petit-a-petit/MIML#6] [Daily Scrum]
.

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - 데이터 베이스 연동
    - 사용자 엔티티 작성
    - 깃허브 Discussion 생성 및 글 게시

- 내일은 어떤 일을 하나요?
    - Spotify 소셜 로그인 과정 학습

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 했나요?
  - 알림 기능 개선 시 기능 비교 방법

- 내일은 어떤 일을 하나요?
  - Spring Event와 transaction 공부

**Comment by Nine-JH** [comment]
* 오늘은 어떤 일을 했나요?
  * 주문 엔티티 작성
  * 주문 및 결제 로직 구상

* 내일은 어떤 일을 하나요?
  * 주문 로직 구현
